src/components/TextBox/index.tsx

```diff
@@ -9,8 +9,8 @@ const Box = styled.p`
   border: solid 2px ${border};
 `
 
-const TextBox = ({ children }: any) => {
-  return <Box>{children}</Box>
+const TextBox = ({ children, ...rest }: { children: React.ReactNode }): React.ReactElement => {
```

Comment:
just curious, how do you plan to type rest property?

Reply:
it's just to pass down the className provided by styled-components, if in any feature time, does anyone wants to pass a new prop beside children, it should be added explicitly in the Props type.

Reply:
but why className can't be explicitly added?